orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/BulkDestroyServerGroupStage.java

```diff
@@ -16,40 +16,49 @@
 
 package com.netflix.spinnaker.orca.clouddriver.pipeline.servergroup;
 
-import com.netflix.spinnaker.orca.clouddriver.pipeline.servergroup.support.TargetServerGroup;
+import com.netflix.spinnaker.kork.dynamicconfig.DynamicConfigService;
 import com.netflix.spinnaker.orca.clouddriver.tasks.MonitorKatoTask;
 import com.netflix.spinnaker.orca.clouddriver.tasks.servergroup.*;
 import com.netflix.spinnaker.orca.kato.pipeline.Nameable;
 import com.netflix.spinnaker.orca.pipeline.StageDefinitionBuilder;
 import com.netflix.spinnaker.orca.pipeline.TaskNode;
 import com.netflix.spinnaker.orca.pipeline.model.Stage;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
 public class BulkDestroyServerGroupStage implements StageDefinitionBuilder, Nameable {
   private static final String PIPELINE_CONFIG_TYPE = "bulkDestroyServerGroup";
 
+  private final DynamicConfigService dynamicConfigService;
+
+  @Autowired
+  public BulkDestroyServerGroupStage(DynamicConfigService dynamicConfigService) {
+    this.dynamicConfigService = dynamicConfigService;
+  }
+
+
   @Override
   public void taskGraph(Stage stage, TaskNode.Builder builder) {
     //TODO(cfieber): how to do locking here...
     // inject an acquire lock stage per distinct cluster in the operation?
     // break into several parallel bulk ops based on cluster and lock/unlock around those?
     // question: do traffic guard checks actually even work in the bulk disable/destroy tasks?
 
-    try {
-      builder
-        .withTask("bulkDisableServerGroup", BulkDisableServerGroupTask.class)
-        .withTask("monitorServerGroups", MonitorKatoTask.class)
-        .withTask("waitForNotUpInstances", WaitForAllInstancesNotUpTask.class)
-        .withTask("forceCacheRefresh", ServerGroupCacheForceRefreshTask.class)
-        .withTask("bulkDestroyServerGroup", BulkDestroyServerGroupTask.class)
-        .withTask("monitorServerGroups", MonitorKatoTask.class)
-        .withTask("waitForDestroyedServerGroup", BulkWaitForDestroyedServerGroupTask.class);
-    } catch (TargetServerGroup.NotFoundException e) {
-      builder
```

Comment:
Believe we long removed the bit that would have actually thrown `TargetServerGroup.NotFoundException`.